ofctrl/fgraphFlow.go

```diff
@@ -899,6 +899,19 @@ func (self *Flow) UnsetDscp() error {
 	return nil
 }
 
+func (self *Flow) AddAction(act Action) error {
+	self.lock.Lock()
+	defer self.lock.Unlock()
+
+	self.flowActions = append(self.flowActions, act)
+
+	if self.isInstalled {
+		self.install()
+	}
+
+	return nil
+}
+
```

Comment:
what about SetTunnelDstIP here, so that we could call flow.SetTunnelDstIP("1.1.1.1") directly

Reply:
ok